src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsExamplesAstConsistencyTest.java

```diff
@@ -88,6 +88,7 @@ public class XdocsExamplesAstConsistencyTest {
     private static final Set<String> SUPPRESSED_EXAMPLES = Set.of(
             "checks/annotation/annotationonsameline/Example2",
             "checks/annotation/missingoverride/Example2",
+            "checks/annotation/missingoverrideonrecordaccessor/Example2",
```

Comment:
Please move it to permanent suppression part of list. With a comment like " No propertes in module, multiple very different examples to ease reading"